controllers/chat/changeChannelDetail.js

```diff
@@ -0,0 +1,44 @@
+const {StreamChat} = require('stream-chat');
+const {CHANNEL_TYPE_STRING} = require('../../helpers/constants');
+const ErrorResponse = require('../../utils/response/ErrorResponse');
+
+/**
+ *
+ * @param {import("express").Request} req
+ * @param {import("express").Response} res
+ */
+const changeChannelDetail = async (req, res) => {
+  const {channel_id, channel_image, channel_name} = req.body;
+  const {userId} = req;
+
+  const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not a good practice to directly use `process.env` in the code. It would be better to have a separate configuration file where you load all your environment variables and then import them wherever needed. This way, if you need to change the source of these values later (for example, from environment variables to some config service), you will only need to change it in one place.

```diff
- const client = new StreamChat(process.env.API_KEY, process.env.SECRET);
+ const { API_KEY, SECRET } = require('../../config');
+ const client = new StreamChat(API_KEY, SECRET);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->